BlueprintUI/Tests/AdaptedEnvironmentTests.swift

```diff
@@ -21,7 +21,7 @@ class AdaptedEnvironmentTests : XCTestCase {
         
         view.layoutIfNeeded()
         
-        XCTAssertEqual(environment?[TestingKey1], "adapted1")
+        XCTAssertEqual(environment?[TestingKey1.self], "adapted1")
```

Comment:
Xcode 12 warns about this; so fixed it.

Reply:
lol I wonder how that ever compiled

Reply:
It seems like it was supported; and now they're pushing away from it? It doesn't warn at all in Xcode 12 AFAIK... it also only (weirdly) works in subscript syntax; it doesn't work in func syntax.